source/beamline/AMPseudoMotorControl.cpp

```diff
@@ -245,7 +245,7 @@ AMControl::FailureExplanation AMPseudoMotorControl::calibrate(double oldValue, d
 		return AMControl::NotConnectedFailure;
 	}
 
-	if (!canMove()) {
```

Comment:
Are we wanting to just remove this `canMove()` check?  Or should your `canCalibrate()` check happen as well?